scriptRunner.py

```diff
@@ -2,7 +2,7 @@
 import sys
 from main import loadEnvFile
 
-loadEnvFile('local-qa', fileString='config/{}.yaml')
+loadEnvFile('local-qa-old', fileString='config/{}.yaml')
```

Comment:
Is this intended or is this a dangling change from some other work?

Reply:
This is intended since I thought this quick ticket was a good opportunity to adjust loadEnvFile for the scriptrunner  module and delete it from the other scripts.